app/routes.tsx

```diff
@@ -350,6 +352,15 @@ export const routes = createRoutesFromElements(
           />
         </Route>
 
+        <Route loader={FloatingIpsPage.loader} element={<FloatingIpsPage />}>
+          <Route path="floating-ips" handle={{ crumb: 'Floating IPs' }} element={null} />
+          <Route
+            path="floating-ips-new"
+            element={<CreateFloatingIpSideModalForm />}
```

Comment:
I think this is missing this, which is why the pools listbox pops in when it should be prefetched.

```
loader={CreateFloatingIpSideModalForm.loader}
```